### PR TITLE
Disable a test that seems to infinitely recurse on a diagnostic somet…

### PR DESCRIPTION
**Pull request by rudkx**
…imes.

Once the associated bug is fixed we can re-enable.

### CODE REVIEW

**Comment by rudkx** [comment]
@swift-ci Please smoke test

File: validation-test/compiler_crashers/28869-swift-diagnosticengine-formatdiagnostictext-llvm-raw-ostream-llvm-stringref-llvm.swift [hunk 1]
```diff
@@ -6,4 +6,5 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
 // RUN: not --crash %target-swift-frontend %s -emit-ir
+// REQUIRES; rdar38932729
```
Comment (jrose-apple) [inline validation-test/compiler_crashers/28869-swift-diagnosticengine-formatdiagnostictext-llvm-raw-ostream-llvm-stringref-llvm.swift]: Typo :-(

Comment (rudkx) [reply validation-test/compiler_crashers/28869-swift-diagnosticengine-formatdiagnostictext-llvm-raw-ostream-llvm-stringref-llvm.swift]: gah, thanks.

**Comment by rudkx** [comment]
@swift-ci Please smoke test

**Pull request merged**